apps/andi/lib/andi_web/router.ex

```diff
@@ -26,7 +25,7 @@ defmodule AndiWeb.Router do
     pipe_through :browser
 
     get "/", Redirect, to: "/datasets"
-    live "/datasets", DatasetLiveView, session: [:path_params]
+    live "/datasets", DatasetLiveView, layout: {AndiWeb.LayoutView, :app}
```

Comment:
What does this do?

Reply:
With the new live view version, if that is not set, then the dataset list page will not include the default andi html layout that includes styling and other things.

Reply:
makes sense